CONTRIBUTING.md

```diff
@@ -149,8 +151,8 @@ The following is an example of a website that *supports* TFA:
 
 If a site does not provide TFA, the `twitter:` field should be included if the site uses
 Twitter. Facebook can also be included using the `facebook` field, as well as Email using
-the `email_address` field. If the website does not use the English language, the `lang:` 
-field should also be included. The fields `sms:`, `email:`, `phone:`, `software:` and 
+the `email_address` field. If the website does not use the English language, the `lang:`
+field should also be included. The fields `sms:`, `email:`, `phone:`, `software:` and
```

Comment:
I think want to keep space here since semantically it's `'lang:' field` and not `'lang:'field`? But it's not the end of the world.

Reply:
The change appears to make no difference once markdown is applied. 

See: [before](https://github.com/brycx/twofactorauth/blob/3de8e300481cbc029ea0bccefc1649aaf0377ea3/CONTRIBUTING.md) _v_ [after](https://github.com/brycx/twofactorauth/blob/dd6c07185e3b834ba5c896f772781ccd4380e005/CONTRIBUTING.md).